phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -911,7 +911,9 @@ private void addCoprocessors(byte[] tableName, TableDescriptorBuilder builder, P
                     QueryServicesOptions.DEFAULT_INDEX_REGION_OBSERVER_ENABLED);
 
             if (tableType == PTableType.INDEX && !isTransactional) {
-                if (globalIndexerEnabled && !newDesc.hasCoprocessor(GlobalIndexChecker.class.getName())) {
+                if (!globalIndexerEnabled && newDesc.hasCoprocessor(GlobalIndexChecker.class.getName())) {
```

Comment:
Would you please rename globalIndexerEnabled as indexRegionObserverEnabled? I have forgotten to rename it

Reply:
I think it should be more generic to reflect newer design parameter.  What do you think?